### PR TITLE
Specify the number of topics for MQTT Demos .

### PR DESCRIPTION
**Pull request by alvin1221**

### CODE REVIEW

**Review by lee-emqx — APPROVED** [review]

File: demo/mqtt/mqtt_client.c [hunk 1]
```diff
@@ -236,7 +236,7 @@ static void
 send_callback (nng_mqtt_client *client, nng_msg *msg, void *arg) {
 	nng_aio *        aio    = client->send_aio;
 	uint32_t         count;
-	uint8_t *        code;
+	reason_code *    code;
```
Comment (wanghaEMQ) [inline demo/mqtt/mqtt_client.c:239]: This issue seems not to be fixed yet

Comment (alvin1221) [reply demo/mqtt/mqtt_client.c:239]: Fixed

**Comment by alvin1221** [comment]
Handle https://github.com/emqx/nanomq/issues/1128

**Review by JaylinYu — COMMENTED** [review]
@wanghaEMQ are you happy about this?

**Review by wanghaEMQ — APPROVED** [review]

**Pull request merged**